perun-rpc/src/main/java/cz/metacentrum/perun/rpc/methods/GroupsManagerMethod.java

```diff
@@ -1140,7 +1160,8 @@ public List<RichUser> call(ApiCaller ac, Deserializer parms) throws PerunExcepti
 						ac.getGroupById(parms.readInt("group")),
 						parms.readList("specificAttributes", String.class),
 						parms.readBoolean("allUserAttributes"),
-						parms.readBoolean("onlyDirectAdmins"));
+						parms.readBoolean("onlyDirectAdmins"),
+						parms.contains("role") ? parms.readString("role") : Role.GROUPADMIN);
```

Comment:
Since `role` is defined as required within the query in openapi, the ternary operator is unnecessary, no?

Reply:
I am not sure if the old method is not used by any script/old GUI and the previous behavior was implemented like the only expected role is GROUPADMIN, so I wanted to let it be available in RPC also in the old form (just for back compatibility) - does it make sense or it is not necessary, what do you think?

Reply:
It doesn't hurt to leave it in. Maybe mention the default value in the javadoc above